deliverables/README.md

```diff
@@ -21,48 +21,117 @@ For projects on Code Academy, or other external exercises, please just paste a l
+
 What are tags?  
-What are attributes?  
-What are forms?  
-What is a div?  
+Tags give specification to elements on where they will be in the code. Tags open with <> and close with </>. A tag of h3 blah blah h3 will be in a different place and have a different meaning than the same text with p tags. 
+
+What are attributes?  Attributes are properties of code that assign a different meaning to a common tag. Ex. p id"serious". The ID makes the paragraph differnt than a normal paragraph.
+
+What are forms? 
+Forms are special elements called controls that are used to make things like check boxes, menus etc. 
+
+What is a div?
+Divs are containers that allow the code to be put into sections on the page. 
+
+What are selectors? 
+Selectors use attributes of an HTML element to select a specific element. Ex: # is a selector indicating ID in CSS.
 
-What are selectors?  
 What are properties?  
+Once an attribute is assigned and the browser processes the code, a node will be created and these nodes have properties. Properties like align, alt, attributes etc. 
+
 What are values?  
-How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
+Values give a hexadecimal value to colors. Instead of red, there is #458393. It gives a value to colors. 
+
+How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?
+The default styles are inherited, ie passed down to a younger generation, from the parent element to the child element. 
+
 What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+You can use text-align or through commands like center me.
 
+What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+You can use in line, external or embedded css. 
 ---
 ### Section 3: Git and Github  
-What is Git?  
+What is Git?
+Git is a system that allows revision control for software development. 
+
 What is SCM?  
+Source code manager. Used to manage source code and track revisions in software.  
+
 What is a VCS?  
-Why is Git useful for a developer?  
+Veritas Cluster Server provides cluster capabilities to systems running other applications. 
+
+Why is Git useful for a developer?
+Git stores and gives feedback to all of your code and allows you to maintain an organized workspace. 
+
 Why is Git useful for a team of developers?  
-How do you create a new Git repository for a project locally?  
+Git allows a team to work together using all of the same files. Editing is done live so everyone can see the changes and add to them. 
+
+How do you create a new Git repository for a project locally? 
+git init command creates a new project. 
+
 How do you create it on Github?  
+You can either use gits built in command line or a regular command line. 
```

Comment:
On github it's gonna be a sequence of clicks.  We're referring to the web interface here.